packages/components/psammead-grid/src/index.jsx

```diff
@@ -0,0 +1,183 @@
+    max: null,
+  },
+};
+
+const group4WrapperMaxWidth = `45.5rem`;
+// (6.75rem * 6) + 5*16px gutters = 728 = 45.5 rem
+
+const group5WrapperMaxWidth = `46.4rem`;
+// (2.95rem * 12) + 11*16px gutters = 742.4 = 46.4 rem
+
+const mediaQuery = ({ min, max, styles }) => {
+  if (min && max) {
+    return `
+      @media (min-width: ${min}) and (max-width: ${max}) {
+        ${styles}
+      }
+    `;
+  }
+  if (min) {
+    return `
+      @media (min-width: ${min}) {
+        ${styles}
+      }
+    `;
+  }
+  if (max) {
+    return `
+      @media (max-width: ${max}) {
+        ${styles}
+      }
+    `;
+  }
+  return '';
+};
+
+const gelGridGutters = css`
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    grid-column-gap: ${GEL_GUTTER_BELOW_600PX};
+  }
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    grid-column-gap: ${GEL_GUTTER_ABOVE_600PX};
+  }
+`;
+
+const gelMargins = css`
+  margin: 0 auto;
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: 0 ${GEL_MARGIN_BELOW_400PX};
+  }
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    padding: 0 ${GEL_MARGIN_ABOVE_400PX};
+  }
+`;
+
+const gelMaxWidths = css`
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
+    max-width: ${group4WrapperMaxWidth};
+  }
+  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
+    max-width: ${group5WrapperMaxWidth};
+  }
+`;
+const group4MaxWidth = css`
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    max-width: ${group4WrapperMaxWidth};
+  }
+`;
+
+const gridMediaQueries = ({ columns, startOffset }) => {
+  const selectedGroups = Object.keys(columns);
+  return selectedGroups.map(group =>
+    mediaQuery({
+      min: groups[group].min,
+      max: groups[group].max,
+      styles: `
+        width: initial;
+        grid-template-columns: repeat(${columns[group]}, 1fr);
+        grid-column-end: span ${columns[group]};
+      ${startOffset ? `grid-column-start: ${startOffset[group]};` : ``}`,
+    }),
+  );
+};
+
+const gridFallbacks = css`
+  ${({ wrapper, columns, parentColumns }) => {
+    if (wrapper || !parentColumns) {
+      return `position: relative;`;
+    }
+    const selectedGroups = Object.keys(columns);
+    return selectedGroups.map(
+      group =>
+        `
+      display: inline-block; 
+      ${mediaQuery({
+        min: groups[group].min,
+        max: groups[group].max,
+        styles: `
+          width: ${(100 * columns[group]) / parentColumns[group]}%;
+          vertical-align: top;
+        `,
+      })}
+    `,
+    );
+  }}
+`;
+
+const GridComponent = styled.div`
+  ${gridFallbacks}
+
+  @supports (display: grid) {
+    ${({ enableGelGutters }) => enableGelGutters && gelGridGutters}
+    ${({ enableGelMargins }) => enableGelMargins && gelMargins}
+    ${({ enableGelMaxWidths }) => enableGelMaxWidths && gelMaxWidths}
+    ${({ enableGroupFourMaxWidth }) =>
```

Comment:
I think `enableGroupFourMaxWidth` shouldnt be an option here, but instead set in a element that expands `Grid`